src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -422,7 +403,9 @@ public PeriodicBackupRunner(DocumentDatabase database, ServerStore serverStore)
                             options.OperateOnTypes |= DatabaseItemType.Tombstones;
 
                         var result = CreateBackup(options, backupFilePath, startDocumentEtag, context);
-                        lastEtag = result.GetLastEtag();
+                        lastEtag = isFullBackup ?
+                            DocumentsStorage.ReadLastEtag(tx.InnerTransaction) :
```

Comment:
Why can't we rely on the last etag from smuggler?

Reply:
Using full backup, we need to get the last db wide etag, after all the current docs, tombstones, etc.